### ISSUE TITLE
508 - Color Contrast issues 

### ISSUE THREAD

**Issue by nickbristow** [18F/calc#96]
![hourglass___home](https://cloud.githubusercontent.com/assets/5296248/7005558/ffdd5884-dc46-11e4-9429-53a707bebb1c.png)

**Comment by nickbristow** [comment]
Color contrast should be 4.5:1 for regular text and 3:1 for large bold text
